tests/topotests/srv6_locator_chunks/test_srv6_locator_chunks.py

```diff
@@ -0,0 +1,187 @@
+#!/usr/bin/env python
```

Comment:
Should we merge this test on tests/topotests/srv6_locator ?

Reply:
I think these topo tests should be separated for simplifying each topo test's responsibility.
the `srv6_locator` topotest guarantees that srv6 manager works correctly on the simplest env.
and `srv6_locator_chunks` topotest interests that srv6 manager behaves correctly as below.

- a proper client is set if freed chunk is re-assigned
- the pool expansion works correctly if manager's chunk pool is temporary exhausted

but for more discussion, I'll keep this thread open.

Reply:
There are very few tests for srv6_locator compared to other topotests. Considering this, I think srv6 developers should focus on tests/topotests/srv6_locator as a unified test for locators. or Only srv6_locator_chunks (anyway, should focus on one instead many)

If they are more complex and require different topology, then maybe the two topotests should be separated, but since they are tests for the same topology and the same topic (srv6 locator), I don't see the point in separating them.

I don't think we should be confused about which topotest to change or both of them when we make future enhancements for srv6 locator.

Reply:
That's make sense for me so I'll merge the `srv6_locator_chunks` topotest into `srv6_locator`, thanks!
so we have two options to merge this, which option do you prefer most? 

- will keep the `srv6_locator` tests, and append `srv6_locator_chunks` tests to it
- will replace the `srv6_locator` tests by multi chunk support.
  - I think the multi-chunk support is a premise in SRv6 context so it has few breaking changes

Reply:
I'll adopt the former for now.